### PR TITLE
Fire b gone now is A B rated and icon fixes

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request
Fire-B-Gone is now now A and B rated
Icons for charge hammers is better
Holywater is better at nulling heat
Fire-B-Gone is better at nulling heat 

## Changelog
:cl:
/:cl:

### CODE REVIEW

File: code/modules/reagents/reagents/core.dm [hunk 1]
```diff
@@ -96,11 +96,41 @@
...
 
+/datum/reagent/water/extinguisher
+	name = "Extinguisher"
+	id = "abwater"
+	description = "A mix of water, with chemicals to reduce heat, oil and suppress fire with heavy particulates."
+	taste_description = "watered down chemicals"
+	glass_icon_state = "glass_clear"
+	glass_name = "water"
+	glass_desc = "The father of all refreshments, this one has floating particulates in it..."
+	nerve_system_accumulations = 50 //Chemical soup
```
Comment (ThePainkiller) [inline code/modules/reagents/reagents/core.dm:110]: If it has NSA make it at least toxic, you wouldn't want to drink extinguisher mix do you....?

File: code/modules/reagents/reagents/core.dm [hunk 2]
```diff
@@ -96,11 +96,41 @@
...
+	fire_supperion_affect = 3 //Three times better at putting out fire then water.
```
Comment (ThePainkiller) [inline code/modules/reagents/reagents/core.dm:111]:
```suggestion
	fire_suppression_effect = 3 //Three times better at putting out fire than water.
```

File: code/modules/reagents/reagents/core.dm [hunk 3]
```diff
@@ -96,11 +96,41 @@
...
+			M.adjustToxLoss(rand(15, 25))
+
+	T.color = "white"
+	return TRUE
+
+
 /datum/reagent/water/holywater
 	name = "Holy Water"
 	description = "A ubiquitous chemical substance that is composed of hydrogen and oxygen with the blessings of faith."
+	fire_supperion_affect = 1.1 //When your hopeful this works...
```
Comment (ThePainkiller) [inline code/modules/reagents/reagents/core.dm:133]:
```suggestion
	fire_suppression_effect = 1.1 //When your hopeful this works...
```

File: code/modules/reagents/reagents/core.dm [hunk 4]
```diff
@@ -134,7 +164,7 @@
 		qdel(hotspot)
 
 	if (environment && environment.temperature > min_temperature) // Abstracted as steam or something
-		var/removed_heat = between(0, volume * WATER_LATENT_HEAT, -environment.get_thermal_energy_change(min_temperature))
+		var/removed_heat = between(0, volume * (WATER_LATENT_HEAT * fire_supperion_affect), -environment.get_thermal_energy_change(min_temperature))
```
Comment (ThePainkiller) [inline code/modules/reagents/reagents/core.dm:167]:
```suggestion
		var/removed_heat = between(0, volume * (WATER_LATENT_HEAT * fire_suppression_effect), -environment.get_thermal_energy_change(min_temperature))
```

**Review by ThePainkiller — CHANGES_REQUESTED** [review]

File: code/modules/reagents/reagents/core.dm (hunk 1 above)
Comment (Trilbyspaceclone) [reply code/modules/reagents/reagents/core.dm:110]: Fire mix iirc is FDA rated as a non-toxic.

**Pull request merged**